build/rofl/oci.go

```diff
@@ -27,6 +28,9 @@ const (
 	ociTypeOrcArtifact = "application/vnd.oasis.orc"
 )
 
+// DefaultOCIRegistry is the default OCI registry.
+const DefaultOCIRegistry = "rofl-orc-registry.prd.oasis.io"
```

Comment:
?

```suggestion
const DefaultOCIRegistry = "oci.oasis.io"
```

Reply:
We also have some `rofl.*` domains?

Reply:
What about `registry.rofl.app`/`orc-registry.rofl.app`?
The `oci.oasis.io` feels too general.

Reply:
`registry.rofl.app` sounds nice. Or is the long-term plan to use `rofl.app` strictly for ROFL apps with open ports? If yes, then `registry.oasis.io`/`orc-registry.oasis.io`/`rofl-registry.oasis.io`?

Reply:
Yes, the plan is to use `rofl.app` for app domains. What about any of the other ones?

Reply:
Some ideas:
* `rofl.sh` (short and similar to `ttl.sh`, but then we dedicate the entire domain; could also be `registry.rofl.sh`)
* `registry.rofl.build`

Reply:
Went forward with `rofl.sh`.